coda-prototype/src/io/StorageUtils.js

```diff
@@ -0,0 +1,196 @@
+class StorageManager {
+    constructor() {
+        /*
+        var manager = this;
+        chrome.storage.local.get("lastEdit", (editObj) => {
+            console.log(editObj);
+            editObj["lastEdit"] = new Date(JSON.parse(editObj["lastEdit"]));
+            console.log(editObj["lastEdit"]);
+            if ( Object.prototype.toString.call(editObj["lastEdit"]) === "[object Date]" ) {
+                if (!isNaN((editObj["lastEdit"]).getTime())) {
+                    // date is in valid format
+                    if (this.isExpired()) {
+                        manager.lastEdit = new Date();
+                        this.clearStorage().then( () => console.log(manager.lastEdit));
+
+                    } else {
+                        manager.lastEdit = editObj["lastEdit"];
+                        console.log(editObj["lastEdit"]);
+                        console.log(manager.lastEdit);
+                    }
+
+                }
+            }
+        });*/
+    }
+    static get instance() {
+        return this._instance || (this._instance = new StorageManager());
+    }
+    isExpired() {
+        // TODO
```

Comment:
This TODO is in the wrong place - this function looks like it does approximately the right thing, so the TODO is to use it?